app/models/liquid_partial_observer.rb

```diff
@@ -0,0 +1,10 @@
+class LiquidPartialObserver < ActiveRecord::Observer
+  def after_save(record)
+    LiquidRenderer::Cache.invalidate
```

Comment:
how does this know to only invalidate when the liquid partial is saved? It seems like it would just invalidate when any record is saved...